src/mappings/DAOTracker/mapping.ts

```diff
@@ -0,0 +1,127 @@
+import { Address, ipfs, store } from '@graphprotocol/graph-ts';
+import {
+  BlacklistDAO,
+  DAOTracker,
+  ResetDAO,
+  TrackDAO,
+} from '../../types/DAOTracker/DAOTracker';
+import {
+  BlacklistedDAO,
+  DAOTrackerContract,
+  ResetDAO as ResetDAOEntity,
+} from '../../types/schema';
+import {
+  Avatar_0_0_1_rc_31,
+  Controller_0_0_1_rc_31,
+  DAOToken_0_0_1_rc_31,
+  Reputation_0_0_1_rc_31,
+} from '../../types/templates';
+import { equalStrings } from '../../utils';
+
+export function getDAOTrackerContract(address: Address): DAOTrackerContract {
+  let daoTracker = DAOTrackerContract.load(address.toHex()) as DAOTrackerContract;
+  if (daoTracker == null) {
+    daoTracker = new DAOTrackerContract(address.toHex());
+    daoTracker.address = address;
+    let daoTrackerSC = DAOTracker.bind(address);
+    daoTracker.owner = daoTrackerSC.owner();
+    store.set('DAOTrackerContract', daoTracker.id, daoTracker);
+  }
+  return daoTracker;
+}
+
+export function handleTrackDAO(event: TrackDAO): void {
+  // Ensure the DAOTrackerContract has been added to the store
+  getDAOTrackerContract(event.address);
+
+  let _avatar = event.params._avatar;
+  let _controller = event.params._controller;
+  let _reputation = event.params._reputation;
+  let _daoToken = event.params._daoToken;
+
+  /* TODO: uncomment when this issue is resolved https://github.com/graphprotocol/graph-node/issues/1333
```

Comment:
this issue is closed

Reply:
I guess we need to have a pre compiled Daos black list.
the subgraph will need to be re indexed to exclude the blacklist dao .(at least on the first phase of the implementation ).

Reply:
Yes we'll need to have the precompiled list instead of checking this at "index time" since we can't view the "latest" version of the registry :/

Reply:
I'll remove this from the code. This also means that we don't need an on-chain mapping of address=>boolean, and we can instead just use the emitted events.

Reply:
Should we remove this from the contract and re-deploy?